### PR TITLE
Update badges.js (using Wisp.nameColor())

### PR DESCRIPTION
**Pull request by FlameOverload**

### CODE REVIEW

File: chat-plugins/badges.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ exports.commands = {
 			userBadges = userBadges.filter(b => b !== selectedBadge);
 			userBadges.push(selectedBadge);
 			Db('userBadges').set(toId(userid), userBadges);
-			if (Users.get(targetUser)) Users.get(userid).popup('|modal||html|<font color="red"><strong>ATTENTION!</strong></font><br /> You have received a badge from <b><font color="' + Wisp.hashColor(toId(user)) + '">' + Tools.escapeHTML(user.name) + '</font></b>: <img src="' + Db('badgeData').get(selectedBadge)[1] + '" width="16" height="16">');
+			if (Users.get(targetUser)) Users.get(userid).popup('|modal||html|<font color="red"><strong>ATTENTION!</strong></font><br /> You have received a badge from ' + Wisp.nameColor(toId(user), true) + '">' + Tools.escapeHTML(user.name) + ': <img src="' + Db('badgeData').get(selectedBadge)[1] + '" width="16" height="16">');
```
Comment (jd4564) [inline chat-plugins/badges.js]: You forgot to remove the `">`

File: chat-plugins/badges.js [hunk 2]
```diff
@@ -83,7 +85,7 @@ exports.commands = {
 			let badgeUserObject = Db('userBadges').object();
 			let users = Object.keys(badgeUserObject);
 			users.forEach(u => Db('userBadges').set(u, (badgeUserObject[u].filter(b => b !== selectedBadge))));
-			this.sendReply("The badge with the name '" + selectedBadge + "' deleted.");
+			this.sendReply("The badge with the name '" + selectedBadge + "'was successfully deleted.");
```
Comment (jd4564) [inline chat-plugins/badges.js]: Missing a space before "was"

**Comment by jd4564** [comment]
Now squash and use a proper commit name.

**Comment by FlameOverload** [comment]
lol k

**Comment by panpawn** [comment]
no, that's the pull request name, not the commit name

and you still haven't squashed

File: chat-plugins/badges.js [hunk 3]
```diff
@@ -69,8 +69,10 @@ exports.commands = {
 			if (!Db('userBadges').has(userid)) return this.errorReply("This user doesn't have any badges.");
 			userBadges = Db('userBadges').get(userid);
 			selectedBadge = parts[2].trim();
+			targetUser = Users.getExact(userid);
 			userBadges = userBadges.filter(b => b !== selectedBadge);
 			Db('userBadges').set(toId(userid), userBadges);
+			if (Users.get(targetUser)) Users.get(userid).popup('|modal||html|<font color="red"><strong>ATTENTION!</strong></font><br /> Your ' + selectedBadge + ' badge have been taken by ' + Wisp.nameColor(toId(user), true));
```
Comment (panpawn) [inline chat-plugins/badges.js:75]: this basically says `if (Users.get(Users.getExact(userid)))`

do you see how that's weird

Comment (supersoniko) [inline chat-plugins/badges.js:75]: Yeah should be just`if (targetUser)`
That's my fault tho; I copied that line from somewhere else without thinking ^^'

**Pull request closed without merging**